.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- 👋 Hi, thanks for sending a PR to LearningTypescript! 💖.
+Please fill out all fields below and make sure each item is true and [x] checked.
+Otherwise we may not be able to review your PR. -->
+
+## PR Checklist
+
+-   [ ] Addresses an existing open issue: fixes #000
+-   [ ] That issue was marked as [accepting prs](https://github.com/LearningTypescript/projects/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
```

Comment:
In practice this typo shouldn't impact anything, but I do like having proper casing 😁 

```suggestion
-   [ ] That issue was marked as [accepting prs](https://github.com/LearningTypeScript/projects/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
```